doc/quickstart/find_data.rst

```diff
@@ -564,6 +564,56 @@ explained in :ref:`extra_facets`, and which content is :download:`available here
 </../esmvalcore/config/extra_facets/ipslcm-mappings.yml>`. These multi-variable
 files must also undergo some data selection.
 
+.. _read_access-esm:
+
+ACCESS-ESM
+^^^^^^^^^^
+
+ESMValTool can read native `ACCESS-ESM <https://research.csiro.au/access/about/esm1-5/>`__
+model output.
+
+.. warning::
+
+  This is the first version of ACCESS-ESM CMORizer for ESMValCore Currently, 
+  only two variables (`tas`, and `pr`) are fully supported.
+
+To read ACCESS-ESM native data, users need to add the directory file structure to 
+`config_developer.yml`.
+
+.. code-block:: yaml
+
+  ACCESS:
+    cmor_strict: false
+    input_dir:
+      default:
+        - '{institute}/{dataset_name}/{exp}/{modeling_realm}/netCDF'
+    input_file:
+      default: '{dataset}.{special_attr}-*.nc'
+    output_file: '{dataset}.{special_attr}'
+    cmor_type: 'CMIP6'
+    cmor_default_table_prefix: 'CMIP6_'
+
+.. hint::
+
+  We only provide one default `input_dir` since this is how ACCESS-ESM native data was 
+  stored on NCI. Users can modify this path to match their local file structure.
+
+  Default `input_file` will automatically load all the native ACCESS-ESM data(1850-2014).
+  If you only need to evaluate a shorter period. Users can add a more specific time range
+  (`{dataset}.{special_attr}-YYYYMM*.nc` e.g.) to avoid the cost of loading unnecessary data.
```

Comment:
The desired time range should never be hardcoded in the config-developer.yml file but set in the recipe using the [`timerange`](https://docs.esmvaltool.org/projects/ESMValCore/en/latest/recipe/overview.html#time-ranges) facet.